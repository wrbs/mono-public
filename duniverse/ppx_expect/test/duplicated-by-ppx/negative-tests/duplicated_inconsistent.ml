module Expect_test_config = struct
  include Expect_test_config

  let upon_unreleasable_issue = `Warning_for_collector_testing
end

let x = ref 0

[%%duplicate
  let%expect_test "what is x" =
    x := !x + 1;
    print_int !x;
    [%expect {| |}]
  ;;]
