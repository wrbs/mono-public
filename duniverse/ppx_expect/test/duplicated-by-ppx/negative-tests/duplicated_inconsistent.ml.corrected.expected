module Expect_test_config = struct
  include Expect_test_config

  let upon_unreleasable_issue = `Warning_for_collector_testing
end

let x = ref 0

[%%duplicate
  let%expect_test "what is x" =
    x := !x + 1;
    print_int !x;
    [%expect
      {|
      (* expect_test: Test ran multiple times with different test outputs *)
      ============================ Output 1 / 2 ============================
      1
      ============================ Output 2 / 2 ============================
      2
      |}]
  ;;]
